code/game/turfs/open/floor.dm

```diff
@@ -114,6 +114,20 @@
 /turf/open/floor/attack_paw(mob/user, list/modifiers)
 	return attack_hand(user, modifiers)
 
+// Click on the floor to close airlocks
+/turf/open/floor/attack_hand(mob/user, list/modifiers)
+	. = ..()
+	if(.)
+		return
+
+	var/mob/living/living = user
+	if(!living)
+		return
+
+	var/obj/machinery/door/airlock/door = locate() in get_turf(src)
+	if(door && !living.combat_mode)
```

Comment:
Does this *really* need to be non-combat mode only? Attacking a floor tile with an empty hand in combat mode doesn't do anything presently to my knowledge.

Reply:
👍 on this, if anything this would be more useful in combat mode when I'm trying to block off a door from someone chasing me, etc